src/account/operations/participation/mod.rs

```diff
@@ -36,6 +37,17 @@ pub struct AccountParticipationOverview {
     participations: HashMap<ParticipationEventId, HashMap<OutputId, TrackedParticipation>>,
 }
 
+/// A participation event with the provided client nodes.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ParticipationEventWithNodes {
+    /// The event id.
+    pub id: ParticipationEventId,
+    /// Information about a voting or staking event.
+    pub data: ParticipationEventData,
+    /// Output participations for events.
+    nodes: Vec<Node>,
```

Comment:
Any specific reason to be private? Just curious

Reply:
No, just copied the other fields where they were already public, will update